backend/users/migrations/0022_user_address_upper_index.py

```
@@ -0,0 +1,59 @@
+and migration state agree on every backend and `makemigrations --check` stays
+clean.
+"""
+
+import django.db.models.functions.text
+from django.db import migrations, models
+
+
+INDEX_NAME = 'users_user_address_upper_idx'
+
+
+def create_index(apps, schema_editor):
```

Ruff (0.16.0)

[warning] Unused function argument: `apps`

(ARG001)